### PR TITLE
Fix: Some typos found using codespell

### PR DESCRIPTION
**Pull request by JMcKiern**

### CODE REVIEW

File: src/script/api/script_event_types.hpp [hunk 1]
```diff
@@ -799,7 +799,7 @@ class ScriptEventTownFounded : public ScriptEvent {
 
 /**
  * Event AircraftDestTooFar, indicating the next destination of an aircraft is too far away.
- * This event can be trigger when the current oder of an aircraft changes, usually either when
+ * This event can be trigger when the current order of an aircraft changes, usually either when
```
Comment (James103) [inline src/script/api/script_event_types.hpp]: "This event can be **trigger** when..." should be "This event can be **triggered** when...".

Comment (JMcKiern) [reply src/script/api/script_event_types.hpp]: Good spot. Thanks

Comment (ldpl) [reply src/script/api/script_event_types.hpp]: I've no idea what's going on here but it's very weird to see such an uncertainty in API description. Is this really such a random event that it's not guaranteed to trigger or the person writing this just wasn't sure? xD

Comment (James103) [reply src/script/api/script_event_types.hpp]: I would rewrite the sentence as "This event is triggered when the current order of an aircraft changes, **and the destination of the new current order is too far away compared to the aircraft's range**.

If the sentence here is kept as is, then there is one more typo: "usually either when loading is done or when **switch** manually" should be "usually either when loading is done or when (it is) **switched** manually" or "usually either when loading is done or when (it is) **changed** manually"

**Comment by nielsmh** [comment]
I wonder if it's really a good idea to retroactively modify the various changelog files, you never know if any tools out there get confused (or upset) if past entries change.

**Comment by LordAro** [comment]
I've definitely modified things in the past, don't think it's a concern

File: src/script/api/script_event_types.hpp (hunk 1 above)
Comment (JMcKiern) [reply src/script/api/script_event_types.hpp]: I just decided to change it to "switched". I wasn't planning on changing the contents of the comments. Thanks anyway.

**Review by nielsmh — APPROVED** [review]

**Comment by nielsmh** [comment]
This should be squashed for merge. Someone else please sanity check it as well, as it's a very large changeset :)

**Review by LordAro — APPROVED** [review]

**Pull request merged**